device/dhcp/servers/v4/dhcpv4serverleases.yaml

```diff
@@ -0,0 +1,91 @@
+components:
+  schemas:
+    DhcpServerV4.Pool:
+      description: >-
+        Configuration for DHCPv4 address pool for a lease.
+      type: object
+      x-status:
+        status: under_review
+      required: [name, start_address]
+      properties:
+        name:
```

Comment:
Is reference to this name required anywhere. Else this should not be needed .

Reply:
we can remove for now